docs/multi-index-search.md

```diff
@@ -23,19 +25,18 @@ multi-index search.
 
 ## How it works
 
-Every search Store instance is in charge of fetching the results from a single
+Every [search store](search-store.md) is in charge of fetching the results from a single
 Algolia index.
 
-To implement a search experience that fetches results from 2 indices,
-you need to have 2 search Stores.
-
-**Tip: you can fetch results from as many indices as you want.**
+To implement a search experience that fetches results from two indices,
+you need to have two search stores.
 
-As a reminder before diving into some code, every time you use an Index
-component (`<ais-index>`), a [search Store is instantiated behind the scenes](search-store-instance.md).
+**Tip: you can fetch results from as many indices as you want. 💥**
 
+As a reminder, before diving into some code, every time you use an [Index
+component](components/index.md), a search store is automatically instantiated.
 
-### Independent Multi-Index searches
+## Independent Multi-Index searches
```

Comment:
I thought it was a mistake because How it works says how it actually works (search store explanation), then the other parts are more or less actual examples (how to use it).